ffi/tests/gen_shared_fns.rs

```diff
@@ -0,0 +1,83 @@
+#![allow(unsafe_code, clippy::restriction)]
+
+use std::{cmp::Ordering, mem::MaybeUninit};
+
+use iroha_ffi::{ffi_bindgen, gen_ffi_impl, handles, FfiResult};
+
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
+pub struct FfiStruct1 {
+    name: String,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
+pub struct FfiStruct2 {
+    name: String,
+}
+
+handles! {0, FfiStruct1, FfiStruct2}
+gen_ffi_impl! {Drop: FfiStruct1, FfiStruct2}
+gen_ffi_impl! {Clone: FfiStruct1, FfiStruct2}
+gen_ffi_impl! {Eq: FfiStruct1, FfiStruct2}
+gen_ffi_impl! {Ord: FfiStruct1, FfiStruct2}
+
+#[ffi_bindgen]
+impl FfiStruct1 {
+    /// New
```

Comment:
I'm curious if this type of description is enough. I'd suggest having something more descriptive